Source/ARTLog.m

```diff
@@ -109,7 +109,7 @@ - (instancetype)init {
 }
 
 - (instancetype)initCapturingOutput:(BOOL)capturing {
-    return [self initCapturingOutput:true historyLines:100];
+    return [self initCapturingOutput:true historyLines:50];
```

Comment:
The history is sent to Sentry when an exception occurs. I think 50 is too low. 100 lines would give us more elements to diagnose an issue. Does this change improve significantly the performance?

Reply:
Will measure and let you know.